### ISSUE TITLE
Needs upgrade to Java 11

### ISSUE THREAD

**Issue by adam-ah** [spring-guides/tut-react-and-spring-data-rest#91]
The parent POM.XML should be updated to:

```
<parent>
        <groupId>org.springframework.boot</groupId>
        <artifactId>spring-boot-starter-parent</artifactId>
        <version>2.1.0.RELEASE</version>
        <relativePath/>
</parent>
```

And the basic POM.XML should be updated to 11

```
<plugin>
<groupId>org.apache.maven.plugins</groupId>
<artifactId>maven-compiler-plugin</artifactId>
<configuration>
<release>11</release>
</configuration>
</plugin>
```

The above only fixes the <i>basic</i> project, the other ones need different fixes too.

**Comment by adam-ah** [comment]
Without this you can run the solution from the IDE, but not from the command line (`mvn spring-boot:run` will fail)

**Comment by mrsarm** [comment]
Just to be clear, as @adam-ah mentioned, the project is not working on machines with Java 11+, and it trigger warnings with Java 9 and Java 10. With the pull request attached, the problem is solved while it continue to use as baseline Java 8, and it's just a dependencies update.

**Comment by gregturn** [comment]
Resolved via a31b2bf.
